Job 60980474 released
Load modules
Lmod has detected the following error: These module(s) or extension(s) exist
but cannot be loaded as requested: "arrow"
   Try: "module spider arrow" to see how to load the module(s).



Create virtual environemnt
Please load a Python module before running virtualenv.
/var/spool/slurmd/job60980474/slurm_script: line 22: /localscratch/tbury.60980474.0/venv/bin/activate: No such file or directory
Install packages
Defaulting to user installation because normal site-packages is not writeable
Looking in links: /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/avx2, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/generic, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic
Requirement already satisfied: pip in /home/tbury/.local/lib/python3.10/site-packages (23.0+computecanada)
Defaulting to user installation because normal site-packages is not writeable
Looking in links: /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/avx2, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/generic, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic
Requirement already satisfied: tensorflow in /home/tbury/.local/lib/python3.10/site-packages (2.11.0+computecanada)
Requirement already satisfied: termcolor>=1.1.0 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (2.2.0)
Requirement already satisfied: libclang>=13.0.0 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (14.0.1+computecanada)
Requirement already satisfied: tensorflow-estimator<2.12,>=2.11.0 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (2.11.0+computecanada)
Requirement already satisfied: tensorboard<2.12,>=2.11 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (2.11.2)
Requirement already satisfied: flatbuffers>=2.0 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (20190709135844+computecanada)
Requirement already satisfied: astunparse>=1.6.0 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (1.6.3+computecanada)
Requirement already satisfied: keras<2.12,>=2.11.0 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (2.11.0+computecanada)
Requirement already satisfied: setuptools in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.10.2/lib/python3.10/site-packages (from tensorflow) (60.7.1)
Requirement already satisfied: grpcio<2.0,>=1.24.3 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (1.47.0+computecanada)
Requirement already satisfied: six>=1.12.0 in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.10.2/lib/python3.10/site-packages (from tensorflow) (1.16.0)
Requirement already satisfied: opt-einsum>=2.3.2 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (3.3.0+computecanada)
Requirement already satisfied: h5py>=2.9.0 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (3.7.0+computecanada)
Requirement already satisfied: numpy>=1.20 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (1.24.2+computecanada)
Requirement already satisfied: tensorflow-io-gcs-filesystem>=0.23.1 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (0.26.0+computecanada)
Requirement already satisfied: typing-extensions>=3.6.6 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (4.5.0+computecanada)
Requirement already satisfied: gast<=0.4.0,>=0.2.1 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (0.4.0+computecanada)
Requirement already satisfied: protobuf<3.20,>=3.9.2 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (3.19.6)
Requirement already satisfied: absl-py>=1.0.0 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (1.4.0+computecanada)
Requirement already satisfied: packaging in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.10.2/lib/python3.10/site-packages (from tensorflow) (21.3)
Requirement already satisfied: wrapt>=1.11.0 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (1.14.1+computecanada)
Requirement already satisfied: google-pasta>=0.1.1 in /home/tbury/.local/lib/python3.10/site-packages (from tensorflow) (0.2.0+computecanada)
Requirement already satisfied: wheel<1.0,>=0.23.0 in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.10.2/lib/python3.10/site-packages (from astunparse>=1.6.0->tensorflow) (0.37.1)
Requirement already satisfied: google-auth<3,>=1.6.3 in /home/tbury/.local/lib/python3.10/site-packages (from tensorboard<2.12,>=2.11->tensorflow) (2.16.1)
Requirement already satisfied: tensorboard-data-server<0.7.0,>=0.6.0 in /home/tbury/.local/lib/python3.10/site-packages (from tensorboard<2.12,>=2.11->tensorflow) (0.6.1+computecanada)
Requirement already satisfied: werkzeug>=1.0.1 in /home/tbury/.local/lib/python3.10/site-packages (from tensorboard<2.12,>=2.11->tensorflow) (2.2.3+computecanada)
Requirement already satisfied: requests<3,>=2.21.0 in /home/tbury/.local/lib/python3.10/site-packages (from tensorboard<2.12,>=2.11->tensorflow) (2.28.2+computecanada)
Requirement already satisfied: tensorboard-plugin-wit>=1.6.0 in /home/tbury/.local/lib/python3.10/site-packages (from tensorboard<2.12,>=2.11->tensorflow) (1.8.1+computecanada)
Requirement already satisfied: google-auth-oauthlib<0.5,>=0.4.1 in /home/tbury/.local/lib/python3.10/site-packages (from tensorboard<2.12,>=2.11->tensorflow) (0.4.6+computecanada)
Requirement already satisfied: markdown>=2.6.8 in /home/tbury/.local/lib/python3.10/site-packages (from tensorboard<2.12,>=2.11->tensorflow) (3.4.1+computecanada)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.10.2/lib/python3.10/site-packages (from packaging->tensorflow) (3.0.7)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /home/tbury/.local/lib/python3.10/site-packages (from google-auth<3,>=1.6.3->tensorboard<2.12,>=2.11->tensorflow) (0.2.8+computecanada)
Requirement already satisfied: cachetools<6.0,>=2.0.0 in /home/tbury/.local/lib/python3.10/site-packages (from google-auth<3,>=1.6.3->tensorboard<2.12,>=2.11->tensorflow) (5.3.0+computecanada)
Requirement already satisfied: rsa<5,>=3.1.4 in /home/tbury/.local/lib/python3.10/site-packages (from google-auth<3,>=1.6.3->tensorboard<2.12,>=2.11->tensorflow) (4.9+computecanada)
Requirement already satisfied: requests-oauthlib>=0.7.0 in /home/tbury/.local/lib/python3.10/site-packages (from google-auth-oauthlib<0.5,>=0.4.1->tensorboard<2.12,>=2.11->tensorflow) (1.3.1+computecanada)
Requirement already satisfied: certifi>=2017.4.17 in /home/tbury/.local/lib/python3.10/site-packages (from requests<3,>=2.21.0->tensorboard<2.12,>=2.11->tensorflow) (2022.12.7+computecanada)
Requirement already satisfied: charset-normalizer<4,>=2 in /home/tbury/.local/lib/python3.10/site-packages (from requests<3,>=2.21.0->tensorboard<2.12,>=2.11->tensorflow) (3.0.1+computecanada)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /home/tbury/.local/lib/python3.10/site-packages (from requests<3,>=2.21.0->tensorboard<2.12,>=2.11->tensorflow) (1.26.14+computecanada)
Requirement already satisfied: idna<4,>=2.5 in /home/tbury/.local/lib/python3.10/site-packages (from requests<3,>=2.21.0->tensorboard<2.12,>=2.11->tensorflow) (3.4+computecanada)
Requirement already satisfied: MarkupSafe>=2.1.1 in /home/tbury/.local/lib/python3.10/site-packages (from werkzeug>=1.0.1->tensorboard<2.12,>=2.11->tensorflow) (2.1.2+computecanada)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /home/tbury/.local/lib/python3.10/site-packages (from pyasn1-modules>=0.2.1->google-auth<3,>=1.6.3->tensorboard<2.12,>=2.11->tensorflow) (0.4.8+computecanada)
Requirement already satisfied: oauthlib>=3.0.0 in /home/tbury/.local/lib/python3.10/site-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib<0.5,>=0.4.1->tensorboard<2.12,>=2.11->tensorflow) (3.2.2+computecanada)
Defaulting to user installation because normal site-packages is not writeable
Looking in links: /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/avx2, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/generic, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic
Requirement already satisfied: scikit-learn in /home/tbury/.local/lib/python3.10/site-packages (1.1.2+computecanada)
Requirement already satisfied: scipy>=1.3.2 in /home/tbury/.local/lib/python3.10/site-packages (from scikit-learn) (1.10.1+computecanada)
Requirement already satisfied: threadpoolctl>=2.0.0 in /home/tbury/.local/lib/python3.10/site-packages (from scikit-learn) (3.1.0+computecanada)
Requirement already satisfied: numpy>=1.21 in /home/tbury/.local/lib/python3.10/site-packages (from scikit-learn) (1.24.2+computecanada)
Requirement already satisfied: joblib>=1.0.0 in /home/tbury/.local/lib/python3.10/site-packages (from scikit-learn) (1.2.0+computecanada)
Defaulting to user installation because normal site-packages is not writeable
Looking in links: /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/avx2, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/generic, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic
Requirement already satisfied: ewstools in /home/tbury/.local/lib/python3.10/site-packages (2.1.0)
Requirement already satisfied: numpy>=1.14.0 in /home/tbury/.local/lib/python3.10/site-packages (from ewstools) (1.24.2+computecanada)
Requirement already satisfied: statsmodels>=0.9.0 in /home/tbury/.local/lib/python3.10/site-packages (from ewstools) (0.13.2+computecanada)
Requirement already satisfied: pandas>=0.23.0 in /home/tbury/.local/lib/python3.10/site-packages (from ewstools) (1.5.3+computecanada)
Requirement already satisfied: scipy>=1.0.1 in /home/tbury/.local/lib/python3.10/site-packages (from ewstools) (1.10.1+computecanada)
Requirement already satisfied: arch>=4.4 in /home/tbury/.local/lib/python3.10/site-packages (from ewstools) (5.3.1)
Requirement already satisfied: plotly>=2.3.0 in /home/tbury/.local/lib/python3.10/site-packages (from ewstools) (5.13.1)
Requirement already satisfied: deprecation>=2.0 in /home/tbury/.local/lib/python3.10/site-packages (from ewstools) (2.1.0+computecanada)
Requirement already satisfied: lmfit>=0.9.0 in /home/tbury/.local/lib/python3.10/site-packages (from ewstools) (1.1.0)
Requirement already satisfied: property-cached>=1.6.4 in /home/tbury/.local/lib/python3.10/site-packages (from arch>=4.4->ewstools) (1.6.4+computecanada)
Requirement already satisfied: packaging in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.10.2/lib/python3.10/site-packages (from deprecation>=2.0->ewstools) (21.3)
Requirement already satisfied: asteval>=0.9.28 in /home/tbury/.local/lib/python3.10/site-packages (from lmfit>=0.9.0->ewstools) (0.9.29)
Requirement already satisfied: uncertainties>=3.1.4 in /home/tbury/.local/lib/python3.10/site-packages (from lmfit>=0.9.0->ewstools) (3.1.7)
Requirement already satisfied: python-dateutil>=2.8.1 in /home/tbury/.local/lib/python3.10/site-packages (from pandas>=0.23.0->ewstools) (2.8.2+computecanada)
Requirement already satisfied: pytz>=2020.1 in /home/tbury/.local/lib/python3.10/site-packages (from pandas>=0.23.0->ewstools) (2022.7.1+computecanada)
Requirement already satisfied: tenacity>=6.2.0 in /home/tbury/.local/lib/python3.10/site-packages (from plotly>=2.3.0->ewstools) (8.2.1)
Requirement already satisfied: patsy>=0.5.2 in /home/tbury/.local/lib/python3.10/site-packages (from statsmodels>=0.9.0->ewstools) (0.5.3)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.10.2/lib/python3.10/site-packages (from packaging->deprecation>=2.0->ewstools) (3.0.7)
Requirement already satisfied: six in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.10.2/lib/python3.10/site-packages (from patsy>=0.5.2->statsmodels>=0.9.0->ewstools) (1.16.0)
Requirement already satisfied: future in /home/tbury/.local/lib/python3.10/site-packages (from uncertainties>=3.1.4->lmfit>=0.9.0->ewstools) (0.18.2+computecanada)
Defaulting to user installation because normal site-packages is not writeable
Looking in links: /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/avx2, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/generic, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic
Requirement already satisfied: matplotlib in /home/tbury/.local/lib/python3.10/site-packages (3.7.0+computecanada)
Requirement already satisfied: kiwisolver>=1.0.1 in /home/tbury/.local/lib/python3.10/site-packages (from matplotlib) (1.4.4+computecanada)
Requirement already satisfied: numpy>=1.21 in /home/tbury/.local/lib/python3.10/site-packages (from matplotlib) (1.24.2+computecanada)
Requirement already satisfied: cycler>=0.10 in /home/tbury/.local/lib/python3.10/site-packages (from matplotlib) (0.11.0+computecanada)
Requirement already satisfied: contourpy>=1.0.1 in /home/tbury/.local/lib/python3.10/site-packages (from matplotlib) (1.0.7+computecanada)
Requirement already satisfied: pyparsing>=2.3.1 in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.10.2/lib/python3.10/site-packages (from matplotlib) (3.0.7)
Requirement already satisfied: fonttools>=4.22.0 in /home/tbury/.local/lib/python3.10/site-packages (from matplotlib) (4.38.0+computecanada)
Requirement already satisfied: pillow>=6.2.0 in /home/tbury/.local/lib/python3.10/site-packages (from matplotlib) (9.4.0+computecanada)
Requirement already satisfied: python-dateutil>=2.7 in /home/tbury/.local/lib/python3.10/site-packages (from matplotlib) (2.8.2+computecanada)
Requirement already satisfied: packaging>=20.0 in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.10.2/lib/python3.10/site-packages (from matplotlib) (21.3)
Requirement already satisfied: six>=1.5 in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.10.2/lib/python3.10/site-packages (from python-dateutil>=2.7->matplotlib) (1.16.0)
-----
 Generate training data 
-----
Traceback (most recent call last):
  File "gen_training_data.py", line 25, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Train classifier of type 1 
-----
Training DL with mtype=1
Traceback (most recent call last):
  File "dl_train.py", line 37, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Train classifier of type 2 
-----
Training DL with mtype=2
Traceback (most recent call last):
  File "dl_train.py", line 37, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Get F1 scores on test data 
-----
Traceback (most recent call last):
  File "dl_test.py", line 13, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Make Sup Fig 2 - example training simulations 
-----
Traceback (most recent call last):
  File "make_fig.py", line 23, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Make Sup Fig 3 - example model test simulations 
-----
Traceback (most recent call last):
  File "make_fig.py", line 15, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Test DL classifier and EWS on Fox model 
-----
Traceback (most recent call last):
  File "test_fox.py", line 29, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
Traceback (most recent call last):
  File "compute_roc.py", line 12, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Test DL classifier and EWS on Westerhoff model 
-----
Traceback (most recent call last):
  File "test_westerhoff.py", line 31, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
Traceback (most recent call last):
  File "compute_roc.py", line 12, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Test DL classifier and EWS on Ricker model 
-----
Traceback (most recent call last):
  File "test_ricker.py", line 32, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
Traceback (most recent call last):
  File "compute_roc.py", line 12, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Test DL classifier and EWS on Kot model 
-----
Traceback (most recent call last):
  File "test_kot.py", line 30, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
Traceback (most recent call last):
  File "compute_roc.py", line 12, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Test DL classifier and EWS on Lorenz model 
-----
Traceback (most recent call last):
  File "test_lorenz.py", line 28, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
Traceback (most recent call last):
  File "compute_roc.py", line 12, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Make Figure 2 - EWS and DL predictions for sample model simulations 
-----
Traceback (most recent call last):
  File "generate_data.py", line 16, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
Traceback (most recent call last):
  File "make_fig.py", line 28, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Make Sup Fig 4 - AUC scores across rof and sigma 
-----
Traceback (most recent call last):
  File "make_fig.py", line 18, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Make Sup Fig 5 - DL favourite bifurcation prop. correct 
-----
Traceback (most recent call last):
  File "make_fig.py", line 17, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Find transition times in chick heart data 
-----
Traceback (most recent call last):
  File "find_transition_times.py", line 12, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Compute EWS in chick heart data 
-----
Traceback (most recent call last):
  File "compute_ews.py", line 15, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Test EWS and DL in chick heart data 
-----
Traceback (most recent call last):
  File "test_chick_heart.py", line 18, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
Traceback (most recent call last):
  File "compute_roc.py", line 15, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Make figure 3 - sample EWS and DL preds in chick heart data 
-----
Traceback (most recent call last):
  File "make_fig.py", line 23, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Make Fig S6 S7 - EWS in chick heart period-doubling 
-----
Traceback (most recent call last):
  File "make_fig.py", line 22, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Make Fig S8 S9 - EWS in chick heart null 
-----
Traceback (most recent call last):
  File "make_fig.py", line 22, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
-----
 Make Figure 4 - ROC curves 
-----
Traceback (most recent call last):
  File "make_fig.py", line 17, in <module>
    import pandas as pd
  File "/home/tbury/.local/lib/python3.7/site-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
pytz: No module named 'pytz'
